crates/sui-rosetta/src/unit_tests/operations_tests.rs

```diff
@@ -44,13 +41,3 @@ async fn test_operation_data_parsing() -> Result<(), anyhow::Error> {
 
     Ok(())
 }
-#[tokio::test]
-async fn test_shorter_bytearray_bug() {
```

Comment:
Why was this removed?

Reply:
@patrickkuo has a fix for what this workaround for originally for and suggested we remove this test altogether as no longer necessary.